src/features/island/hud/components/inventory/Basket.tsx

```diff
@@ -30,10 +33,14 @@ import { BEANS } from "features/game/types/beans";
 export const ITEM_CARD_MIN_HEIGHT = "148px";
 export const TAB_CONTENT_HEIGHT = 400;
 
+interface Props {
+  state: GameState;
+}
+
 const isSeed = (selectedItem: InventoryItemName) =>
   selectedItem in CROP_SEEDS();
 
-export const Basket: React.FC = () => {
+export const Basket: React.FC<Props> = ({ state }: Props) => {
```

Comment:
Looks like this component already has access to the state without needing to pass this is as a prop.


You should just be able to pull the necessary values off the `gameState.context.state`
```
const { inventory, bumpkin, collectibles } = gameState.context.state;
```

Reply:
how did i miss that? 😥